pkg/binary/binary_test.go

```diff
@@ -94,7 +94,7 @@ func GetTestHeaders() []TestHeader {
 
 // writeTestBinary Writes a test binary and returns the path
 func writeTestBinary(t *testing.T, base64Data *string) *os.File {
-	f, err := ioutil.TempFile(os.TempDir(), "test-binary-")
+	f, err := ioutil.TempFile("", "test-binary-")
```

Comment:
Should we use https://golang.org/pkg/os/#CreateTemp?

Reply:
Yes, fixed as suggested :+1: